src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.de.json

```diff
@@ -13,5 +13,7 @@
     "sulu_security.edit": "Bearbeiten",
     "sulu_security.delete": "Löschen",
     "sulu_security.live": "Live",
-    "sulu_security.security": "Sicherheit"
+    "sulu_security.security": "Sicherheit",
+    "sulu_security.first_name": "Vorname",
+    "sulu_security.last_name": "Nachname"
```

Comment:
These two translations should probably be taken from the `ContactBundle`, since this bundle is dependant on it anyway, and they should not differ.